test/core/tsi/alts/handshaker/alts_tsi_handshaker_test.cc

```diff
@@ -328,6 +341,25 @@ static void on_client_next_success_cb(tsi_result status, void* user_data,
                     peer_account.size) == 0);
   GPR_ASSERT(memcmp(ALTS_TSI_HANDSHAKER_TEST_LOCAL_IDENTITY, local_account.data,
                     local_account.size) == 0);
+  size_t iter = UPB_MAP_BEGIN;
+  grpc_gcp_AltsContext_PeerAttributesEntry* peer_attributes_entry = 
+      grpc_gcp_AltsContext_peer_attributes_nextmutable(ctx, &iter);
+  GPR_ASSERT(peer_attributes_entry != nullptr);
+  while ( peer_attributes_entry != nullptr) {
```

Comment:
nit: remove the empty space in front of `peer_attributes_entry`. Ditto elsewhere.

Reply:
Still having some issues with the expected checks they do not seem to run even when I commit?